templates/files/partials/file_list.html

```diff
@@ -0,0 +1,26 @@
+{# Template fragment for display a file list with ability to add and remove #}
+{# Required template vars: #}
+{#  - upload_url: URL to upload a file #}
+{#  - See also "files/partials/file_item.html" #}
+<div x-data="{}">
```

Comment:
Should this be here?

Reply:
Yes, it is a signal to Alpine to process the element: https://alpinejs.dev/directives/data#data-less-components